Docker/python-nginx/python3.6-alpine3.7/dockerrun.sh

```diff
@@ -91,7 +91,6 @@ if [[ -z $DD_ENABLED ]]; then
   run uwsgi --ini /etc/uwsgi/uwsgi.ini
 ) &
 else
-pip install ddtrace
```

Comment:
for `Docker/python-nginx/python3.6-alpine3.7` you didn't add ddtrace to the Dockerfile, is that on purpose?

Reply:
No that is an oversight. I will add now, thanks!